segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/TruncateOperation.java

```diff
@@ -90,9 +90,8 @@
                                         return commit(txn)
                                                 .handleAsync(this::handleException, chunkedSegmentStorage.getExecutor())
                                                 .thenComposeAsync(vv ->
-                                                                chunkedSegmentStorage.collectGarbage(chunksToDelete).thenApplyAsync(vvv -> {
+                                                                chunkedSegmentStorage.collectGarbage(chunksToDelete).thenRunAsync(() -> {
```

Comment:
FYI, Java allows you to compress this syntax into `thenRunAsync(this::postCommit, executor)`

Reply:
fixed.